docs/src/app/components/pages/components/dialog.jsx

```diff
@@ -25,7 +25,7 @@ class DialogPage extends React.Component {
       '//Standard Actions\n' +
       'var standardActions = [\n' +
       '  { text: \'Cancel\' },\n' +
-      '  { text: \'Submit\', onClick: this._onDialogSubmit, ref: \'submit\' }\n' +
+      '  { text: \'Submit\', onTouchTap: this._onDialogSubmit, ref: \'submit\' }\n' +
```

Comment:
Not sure if we still need to document onClick here - passing `onTouchTap` always sounds like a better idea to me.